style.css

```diff
@@ -0,0 +1,52 @@
+
+main{
+    background-color: #eee;
+    margin: 10px;
+    padding: 10px;
+    border-radius: 5px;
+}
+h1{
+text-align: center;
+}
```

Comment:
kindly consider using class or id selectors instead of tag selectors. Because of CSS best practices and to facilitate the code maintenance...